packages/fxa-settings/src/pages/ResetPasswordWithRecoveryKeyVerified/index.stories.tsx

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import ResetPasswordWithRecoveryKeyVerified from '.';
+import AppLayout from '../../components/AppLayout';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+import { useNavigate } from '@reach/router';
+
+export default {
+  title: 'pages/ResetPasswordWithRecoveryKeyVerified',
```

Comment:
Do we want to call these pages `Confirmed` or `Verified`? At first I thought "confirmed" would be better, but I'm thinking using `Verified` might actually be nicer since it reflects the URL.

Also, do we have another ticket filed to move our Settings pages into `pages` yet? 👀

Reply:
Shoot you're right, we do want to go with `Confirmed` -- and are our Settings pages going to move into `pages`? I don't remember that discussion at all 🤦

Reply:
I think of those pages as all living together separately since they have a distinctive shared context. Would it be possible to push that work further forward so as to not add scope to the present work?

Reply:
Moving Settings components/pages should definitely be a separate ticket, but I don't think it's filed yet. I'm not sure we'd actually decided if those would be nested (ex. `src/pages/Settings/ChangePassword`), and that seems like a conversation that's out of scope for now.

For now, though, we all agree to create the new pages in `/src/pages/` and not include `Page` in the folder/page name?

Reply:
> Hm we did want to go with `Confirmed`, but yeah that's the url....I'll check w Vesta. Are our Settings pages going to move into `pages`? I don't remember that discussion at all 🤦 😬 Should we have a meeting to discuss?

I'm not sure Vesta makes the call here since this is back-end stuff to her. 😬 I think I prefer matching the URL but I'm OK with either as long as we're consistent.

That discussion was part of [that Slack discussion](https://mozilla.slack.com/archives/CLV3KMZ8B/p1668462473186869) a few weeks ago when we were discussing `pages/` but yeah we never finalized anything, I'm just calling this out since it seems preferred to stick these new pages under `pages/` as the first couple are already done this way so I updated the Storybook epic acceptance criteria to match that as well, since I'd previously noted to use `PageWhatever`. 

I personally don't like the idea of new content-server pages going under `pages/` while Settings follows the `PageWhatever` format, so yeah we should have a separate ticket filed for Settings pages and another separate one for admin-panel to follow the same pattern.

> I'm not sure we'd actually decided if those would be nested (ex. `src/pages/Settings/ChangePassword`), and that seems like a conversation that's out of scope for now.

To be fair, I'm not sure we officially "decided" to use `pages/` either and we just did it 😆 I think I personally like `pages/settings/NameOfPage` to match our `components/` hierarchy, but we can always continue discussions on that thread too.

> For now, though, we all agree to create the new pages in `/src/pages/` and not include `Page` in the folder/page name?

Yeah I'm fine with this. We should follow up in that thread though and probably send the message to the team chat as well about it so everyone's aware.

Reply:
👍 Sounds good, thanks for the clarification! Could we file a spike to figure out how to transform the Settings app to match this pattern?

Reply:
@millsoper @vpomerleau What's your opinion on `pages/Settings/NameOfPage`? If all 3 of us like it, I think we can just go with it and let the team know similar to the `pages/` decision, since no one seemed to have strong opinions on that thread. If we're in disagreement yeah let's file a spike or talk in team chat but if we all agree, I think we can just file a task.

Reply:
I've filled issues on GitHub, and did suggest `pages/Settings/NameOfPage`:

settings: https://github.com/mozilla/fxa/issues/14650
admin-panel: https://github.com/mozilla/fxa/issues/14651

These aren't syncing to Jira for some reason

Reply:
@vpomerleau Thank you! 🙌